deployment/terraform/ssh/ssh.go

```diff
@@ -77,6 +79,19 @@ func (sshc *Client) Upload(src io.Reader, sudo bool, dst string) error {
 	return sess.Run(cmd)
 }
 
+// UploadFile uploads a given file path to a given destination file.
+func (sshc *Client) UploadFile(src string, sudo bool, dst string) error {
```

Comment:
Did you order the args on purpose like that? Wouldn't it look clearer if `sudo` was kept last? 
This applies to the `ssh` method too.

Reply:
That's true actually. Will change.

Reply:
done